### ISSUE TITLE
Default to AWS region and profile from standard cli config

### ISSUE THREAD

**Issue by adrian-skybaker** [binxio/aws-ssh-tunnel#5]
It would be great if rather than requiring a separate config file, you could simply inherit the current configuration of the CLI (profile, region etc).

**Comment by DaanRademaker** [comment]
+1

**Comment by DaanRademaker** [comment]
Or as a parameter would also be very useful.

**Comment by DaanRademaker** [comment]
I think it would be more sensible to either have no overwrites and let boto3 look at your AWS_DEFAULT_PROFILE and AWS_DEFAULT_REGION variables. Or add parameters for these in the cli tool. The config file in the package is not so handy for parameters that change.

**Comment by mvanholsteijn** [comment]
@DaanRademaker you are absolutely right. At the moment, I do not have time to fix this. PRs are welcome, and otherwise I can pick this up in two weeks time.

**Comment by mvanholsteijn** [comment]
@DanielRieske  would you be able to pick up this.

The rational way would be if AWS_PROFILE and/or AWS_REGION is not set in the configuration for this utility, it won't set it and uses the default AWS SDK settings instead of prompting.

**Comment by DaanRademaker** [comment]
I might have some time to create pull request!

**Comment by DaanRademaker** [comment]
What would be the preferred override of for example profile?
I was thinking:

1. option (--profile)
2. config (as it is currently implemented)
3. fallback to AWS_DEFAULT_PROFILE environment variable?

**Comment by DanielRieske** [comment]
@DaanRademaker Let me know if there is anyway I can assist, I also have some time at hand.

For now, I would let `--profile` take precendence over everything, `config` being the default and using `AWS_DEFAULT_PROFILE` to override the profile in `config`

**Comment by mvanholsteijn** [comment]
If the aws-ssh-tunnel config file does not contain a profile or region, it will use the AWS SDK defaults when creating the aws.Session.

The --profile and --region override the aws-ssh-tunnel config.

**Comment by DaanRademaker** [comment]
Pull request with the changes.
https://github.com/binxio/aws-ssh-tunnel/pull/10

**Comment by DaanRademaker** [comment]
@DanielRieske will we do a release soon? This issue is solved but main has not been released yet.

**Comment by DaanRademaker** [comment]
Aah yes sounds good. If you want I can help setup a more pythonic way to build and deploy the package (using poetry). We for example implemented that already in repo of your sister company (xebia data) [here](https://github.com/godatadriven/databricks-cdk). This way you can just release patch, minor major from github action.
